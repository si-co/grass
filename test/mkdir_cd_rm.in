login Acidburn
pass CrashOverride

# cd to nonexisting dir
cd tmp

# cd to existing dir
mkdir tmp
cd tmp

# cd to nonexisting dir
cd tmp2

# cd to existing dir
mkdir tmp2 
cd tmp2

# cd up and down several levels
cd ../..
cd tmp/tmp2

# cd out of bounds (2x)
cd ../../..
cd ..
cd ..
cd ..

# rm nonexisting directory
rm tmp2

# rm existing directories
cd tmp
rm tmp2
cd ..
rm tmp

# cd deleted directory
cd tmp

exit
