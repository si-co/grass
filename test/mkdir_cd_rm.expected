
# cd to nonexisting dir
Error : directory does not exist

# cd to existing dir

# cd to nonexisting dir
Error : directory does not exist

# cd to existing dir

# cd up and down several levels

# cd out of bounds
Error : invalid path
Error : invalid path

# rm nonexisting directory
Error : directory does not exist

# cd deleted directory
Error : directory does not exist
